src/test/java/com/checkout/workflows/four/WorkflowReflowTestIT.java

```diff
@@ -1,18 +1,19 @@
 package com.checkout.workflows.four;
 
 import com.checkout.payments.four.response.PaymentResponse;
+import com.checkout.workflows.four.actions.response.WorkflowInvocationsResponse;
 import com.checkout.workflows.four.events.SubjectEvent;
 import com.checkout.workflows.four.events.SubjectEventsResponse;
 import com.checkout.workflows.four.reflow.ReflowByEventsRequest;
 import com.checkout.workflows.four.reflow.ReflowBySubjectsRequest;
-import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
 import java.util.Collections;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class WorkflowReflowTestIT extends AbstractWorkflowTestIT {
```

Comment:
Why this test in `WorkflowReflowTestIT`? Can you please create a different test class for this? For example `WorkflowActionsIntegrationTest`.